1 Foundations/isArray.js

```diff
@@ -1,4 +1,15 @@
-// This tells is the value is an array or not
+//Array.isArray()
+// The Array.isArray() method determines whether the passed value is an Array or not.
+
+// Resource: http://web.mit.edu/jwalden/www/isArray.html
+
+// syntax: Array.isArray(value)
+
+// Parameters:
+//            value means the value to be checked
+
 (function () {
     console.log(Array.isArray(arguments))
 })()
+
+// Return value: true if given parameter is an array otherwise it return false
```

Comment:
The explanations look good! 👍🏻
Please look into [SampleJS.md](https://github.com/MadaZZ/JS-Concepts/blob/master/SampleJS.md) for file format